test-suite-groovy/src/test/groovy/io/micronaut/docs/inject/generics/VehicleSpec.groovy

```diff
@@ -12,5 +12,6 @@ class VehicleSpec extends Specification {
     void 'test start engine'()  {
         expect:
         vehicle.start() == 'Starting V8'
```

Comment:
Why is here V8?

Reply:
the package has a V8 and a V6 class https://github.com/micronaut-projects/micronaut-core/tree/3.5.x/inject-java/src/test/groovy/io/micronaut/inject/generics

We were display the V6 class in the docs but we had no test for it returning 6 cylinders.